src/lib/store/modules/entity.store.js

```diff
@@ -55,9 +55,9 @@ const actions = {
             ext: entity.ext
         });
     },
-    async createEntity(obj, {clientId, entityId, name, description, type, ownerId, fullText, binaryData}) {
-        return await custosService.entities.createEntity({
```

Comment:
Reinclude this return statement. It's been currently used.

Reply:
I checked the code base, we are only using the `createEntity` function only in the `TenentNewEntity` component. If we want to avoid the `parentId`  param we could send undefined which will replicate the same functionality.

Reply:
Eg: -The custos portal source is currently being used as an npm dependency for EMC portal where this is currently needed. 
https://github.com/SciGaP/emcenter-gateway/blob/master/package.json#L12
https://github.com/SciGaP/emcenter-gateway/blob/master/src/pages/storage/storage-preference-new.vue#L131

Reply:
The parameter for Dispatch is `Object` and inside the dispatch, we are creating another Object based on the parameter object more like `...` spread operation. If the passed object doesn't have `parentId` it will automatically assign to `undefined` thus our functionality in both cases won't effect.

Reply:
Krishna. The concern is that EMC portal need to have this return even though it's not relevant to custos portal much

Reply:
I will add a return statement for `createEntity` method.